transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -55,8 +55,11 @@
     // This number will be split evenly among all EventLoopGroup children
     // ONLY CHANGED FOR TESTING!
     static Integer aioMaxConcurrency = Integer.getInteger("netty.aio.maxConcurrency", 128);
+    // Represents the minimum number of outstanding aio requests per loop, we pick the maximum
+    // between this value and (aioMaxConcurrency / nloops)
+    static Integer aioPerLoopMaxConcurrency = Integer.getInteger("netty.aio.perLoopMaxConcurrency", 1 << 6); // 64
```

Comment:
We can pick a smaller number as it doesn't make much sense with only 128 as a maximum value. However, I've been testing with 128 events per core without adverse results. The kernel allocates `n_cores x 8` events per AIO context. We should look into reducing the number of AIO contexts that we use, for this we have [DB-1463](https://datastax.jira.com/browse/DB-1463).

Reply:
If you've verified it then I'm happy to go with your number.  

Scylla seems to run a test script that finds the optimal settings per machine.

> 2017-12-06 22:18:06,032 [FJP:12]             INFO  server-1     - STDOUT: Recommended --max-io-requests: 68
> 2017-12-06 22:18:06,034 [FJP:12]             INFO  server-1     - STDOUT: Recommended --num-io-queues: 17